GHzDACs/ghz_fpga_server.py

```diff
@@ -662,23 +665,24 @@ def run(self, runners, reps, setupPkts, setupState, sync, getTimingData,
                         # collected.
                         # If this fails, something BAD happened!
                         r = yield waitPkt.send()
-                        logging.info("waitPkt sent")
+                        logging.info('waitPkt sent')
                         try:
                             # Then set up
-                            logging.info("sending setupPkts...")
+                            logging.info('sending setupPkts...')
                             yield self.sendAll(setupPkts, 'Setup')
-                            logging.info("...setupPkts sent")
+                            logging.info('...setupPkts sent')
                             self.setupState = setupState
                         except Exception as e:
                             # if there was an error, clear setup state
                             logging.info('catching setupPkts exception')
                             self.setupState = set()
-                            logging.error("Exception in setupPkts: %s" % str(e))
+                            logging.error(
+                                    'Exception in setupPkts: %s' % str(e))
```

Comment:
We're definitely inconsistent on this, but we want to move to `.format` instead of `%` formatting